google/resource_compute_subnetwork.go

```diff
@@ -68,6 +77,26 @@ func resourceComputeSubnetwork() *schema.Resource {
 				Optional: true,
 			},
 
+			"secondary_ip_range": &schema.Schema{
```

Comment:
I'm not sure if it will ever make a difference, but this should probably be marked `ForceNew` as well as it's children.